src/VoxaApp.ts

```diff
@@ -16,6 +16,7 @@ const log: debug.IDebugger = debug("voxa");
 
 export interface IVoxaAppConfig extends IRendererConfig {
   appIds?: string[]|string;
+  defaultFulfillIntents?: string[];
```

Comment:
as this is platform specific, let's move it to the AlexaPlatform constructor,  something like this https://github.com/mediarain/voxa/blob/v3/src/platforms/botframework/BotFrameworkPlatform.ts#L50